### PR TITLE
Article/witwa nominations

### PR DESCRIPTION
**Pull request by NatalieHarmsworth**
WiTWA nomination post. 
Voting opened today/

### CODE REVIEW

File: _posts/2020-09-07-witwa-nominations.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+author: Natalie Laing
+---
+
+![Mechanical Rock Ladies](/img/MechRockLadies.png)
+
+<br/>
+
+## My story
+
+My experience has been like that of so many women in the tech industry. At university, I could count the number of women on my course on one hand. At my first job I was the only female on a team of twelve developers.When I joined Mechanical Rock I was employee number fifteen and again I was the only woman. I thought this was normal.
```
Comment (Hamo69) [inline _posts/2020-09-07-witwa-nominations.md:15]: developers.When

should be 

developers. When

File: _posts/2020-09-07-witwa-nominations.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
```
Comment (Hamo69) [inline _posts/2020-09-07-witwa-nominations.md:2]: Is it possible to add a photo in and reference it in the title?

**Review by Hamo69 — APPROVED** [review]

File: _posts/2020-09-07-witwa-nominations.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+![Mechanical Rock Ladies](/img/MechRockLadies.png)
+
+<br/>
+
+## My story
+
+My experience has been like that of so many women in the tech industry. At university, I could count the number of women on my course on one hand. At my first job I was the only female on a team of twelve developers.When I joined Mechanical Rock I was employee number fifteen and again I was the only woman. I thought this was normal.
+
+Over the past 2 and a half years I have seen Mechanical Rock grow both in size and diversity. We have hired some amazingly talented women and I have loved seeing how the company has evolved as each of these ladies have brought something so new and wonderful to our team. In fact, my current project is majority female and I love that I get to pair with other talented female developers every day.  
+I can honestly say that it makes a difference, and it has made Mechanical Rock an even more amazing place to work.
```
Comment (Hamo69) [inline _posts/2020-09-07-witwa-nominations.md:18]: Perhaps this could be bolded / highlighted?

Comment (NatalieHarmsworth) [reply _posts/2020-09-07-witwa-nominations.md:19]: The last sentence?

File: _posts/2020-09-07-witwa-nominations.md (hunk 2 above)
Comment (NatalieHarmsworth) [reply _posts/2020-09-07-witwa-nominations.md:2]: I just added it as the image we are using in the post.

File: _posts/2020-09-07-witwa-nominations.md (hunk 3 above)
Comment (Hamo69) [reply _posts/2020-09-07-witwa-nominations.md:19]: This bit
"I can honestly say that it makes a difference, and it has made Mechanical Rock an even more amazing place to work."

Comment (NatalieHarmsworth) [reply _posts/2020-09-07-witwa-nominations.md]: Ok I have made that bold

**Review by Hamo69 — APPROVED** [review]

**Pull request merged**